src/notation/view/notationcontextmenumodel.cpp

```diff
@@ -202,6 +204,13 @@ MenuItemList NotationContextMenuModel::makeElementItems()
     if (hitElement && hitElement->isEditable()) {
         items << makeSeparator();
         items << makeMenuItem("edit-element");
+        if (hitElement->isHairpinSegment() && isSingleSelection()) {
```

Comment:
@avvvvve 
We usually add unique items for an element to the end of the list. For example, for StaffText, SystemText... ChangeInstrument, HorizontalBox/VerticalBox

Reply:
@Eism @mathesoncalum Good catch, yes let's put it after Style instead! I'll change the design to match.

Reply:
(Done!)